src/main/resources/scripts/equip anything.sk

```diff
@@ -2,20 +2,20 @@
 # Allows specified items to be used as helmets for survival players (Minecraft 1.9 recommended).
 #
 
-aliases:
-	custom helmets = iron block, gold block, diamond block # Change this to anything you'd like
+init:
+	set {helmets::*} to iron block, gold block and diamond block
```

Comment:
Scriptt aliases should be preferred over random global variables. They're gone when the script is removed.

Reply:
I tested those, acted strangely. They gave errors about how ID's can't be used in newer versions, even though I didn't use ID's.

Reply:
Could you send the code you used please?